docs-public/snippets.js

```diff
@@ -282,10 +282,20 @@ function bootstrapAngular(code, resize) {
         code,
         "import { platformBrowserDynamic } from '@angular/platform-browser-dynamic';",
         "import { NgModule } from '@angular/core';",
+        "import Raven = require('raven-js');",
+        "import { ErrorHandler } from '@angular/core';",
+        "console.log(Raven, Raven.config);",
```

Comment:
Leftover `console.log`.

Also, the imports from `@angular/core` can be merged.
`import { NgModule, ErrorHandler } from '@angular/core'`